github/ci/prow/files/jobs/macvtap-cni/macvtap-cni-postsubmits.yaml

```diff
@@ -0,0 +1,24 @@
+postsubmits:
+  kubevirt/macvtap-cni:
```

Comment:
We gotta limit it to master. Otherwise minor-branch merged will override master.

Reply:
Done.